### PR TITLE
Send confirmation SMS to requestee after registering a request

### PR DESCRIPTION
**Pull request by arunkrishnan**
Need to update the following environment variables before deploying
SMS_API, SMS_USER, SMS_PASSWORD

### CODE REVIEW

File: mainapp/views.py [hunk 1]
```diff
@@ -49,6 +52,11 @@ class CreateRequest(CreateView):
     ]
     success_url = '/req_sucess/'
 
+    def form_valid(self, form):
+        self.object = form.save()
+        send_confirmation_sms(self.object.requestee_phone)
```
[the remark this answers is not in the document]
Comment (aswins) [reply mainapp/views.py]: Can we consider this? https://python-rq.org/ Will have to have a worker deployed for this.

File: mainapp/sms_handler.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    )
+    api_url = "{}?username={}&password={}&message={}&numbers={}&senderid=KSITMK".format(
+        SMS_ENDPOINT,
+        SMS_USER,
+        SMS_PASSWORD,
+        confirmation_message,
+        phone_number.lstrip("0"),
+    )
+    try:
+        requests.get(api_url)
```
Comment (ashwinrajeev) [inline mainapp/sms_handler.py]: set timeout

File: mainapp/redis_queue.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import os
+
+from redis import Redis
+from rq import Queue
+
+redis_host = os.environ.get("REDIS_URL")
```
Comment (ashwinrajeev) [inline mainapp/redis_queue.py]: REDIS_URL format is redis://username:password@localhost:6379

```
import redis
r = redis.from_url(os.environ.get("REDIS_URL"))
```

**Comment by syam3526** [comment]
@biswaz any updates on this

**Pull request merged**